libmscore/dynamic.cpp

```diff
@@ -373,7 +373,19 @@ QVariant Dynamic::propertyDefault(P_ID id) const
 
 QString Dynamic::accessibleInfo() const
       {
-      return QString("%1: %2").arg(Element::accessibleInfo()).arg(this->dynamicTypeName());
+      QString s;
+
+      if (dynamicType() == Dynamic::Type::OTHER) {
```

Comment:
Would it be smart to override screenReaderInfo then? because the elision will not be nice when said by a screen reader.